samples/mcpclient/appPackage/manifest.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.15/MicrosoftTeams.schema.json",
+  "version": "1.0.0",
+  "manifestVersion": "1.15",
```

Comment:
is this being checked in for use with TTK? Should the other TTK yml/bicep files be accompanying it?

Reply:
Nope. Removing.